lib/src/clojure_lsp/feature/completion.clj

```diff
@@ -475,6 +485,12 @@
        (mapv #(dissoc % :priority))
        not-empty))
 
+(defn ^:private limiting-items
+  "Limit the returned items for better performance.
+  If user needs more items one should be more specific in the completion query."
+  [size items]
+  (drop-last (- (count items) size) items))
```

Comment:
This looks a lot like `(take size items)`. Are there performance differences?

Reply:
Uhh, completely forgot take here :facepalm: I'll change it